### PR TITLE
Add dogtags $PBOPREFIX$ and Improve function headers

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Add `$PBOPREFIX$`
- Improve function headers with examples
- Use `QQGVAR` macro instead of `QUOTE(QGVAR())`
- Add @esteldunedain do maintainers
- Update CfgPatches with new author stuff
- Add beautified component name

How was that missed... you know who you are...

### CODE REVIEW

**Comment by esteldunedain** [comment]
Thanls

File: addons/dogtags/Dogtag.hpp [hunk 1]
```diff
@@ -4,8 +4,8 @@ class RscStructuredText;
 class RscTitles {
     class GVAR(singleTag) {
         idd = -1;
-        onLoad = QUOTE(uiNamespace setVariable [ARR_2(QUOTE(QGVAR(tag)),_this select 0)]);
-        onUnload = QUOTE(uiNamespace setVariable [ARR_2(QUOTE(QGVAR(tag)),nil)]);
+        onLoad = QUOTE(uiNamespace setVariable [ARR_2(QQGVAR(tag),_this select 0)]);
```
Comment (commy2) [inline addons/dogtags/Dogtag.hpp]: about this. you could've also wrote:
`onLoad = QUOTE(uiNamespace setVariable [ARR_2('GVAR(tag)',_this select 0)]);`

Comment (jonpas) [inline addons/dogtags/Dogtag.hpp]: Minor difference, QQ keeps highlighting nice though.

Comment (commy2) [inline addons/dogtags/Dogtag.hpp]: ok

Comment (SilentSpike) [inline addons/dogtags/Dogtag.hpp]: Could also have wrote:

`onLoad = QUOTE(SETUVAR(QGVAR(tag),_this select 0));`

Comment (jonpas) [inline addons/dogtags/Dogtag.hpp]: I didn't write the original code (_runs away_)! Although I dislike those set/getVariable macros.

**Pull request merged**